tests/test_efficiency.py

```diff
@@ -30,6 +41,31 @@ def test_efficiency_simple_values():
     assert approx(high_uncertainty) == np.array([0.03056316])
     assert approx(low_uncertainty) == np.array([0.03056316])
 
+    assert approx(high_uncertainty[0], 0.02) == simple_efficiency_uncertainty(n1, n2) # 0.02 relative error
+    assert approx(low_uncertainty[0], 0.02) == simple_efficiency_uncertainty(n1, n2) # 0.02 relative error
+
+    # Test with larger numbers
+    n1 = 10000000
+    n2 = 1000000
+
+    h1 = make_hist(data=[1] * n1, bins=1, range=(0, 2))
+    h2 = make_hist(data=[1] * n2, bins=1, range=(0, 2))
+
+    values, high_uncertainty, low_uncertainty = get_comparison(
+        h2, h1, comparison="efficiency"
+    )
+
+    assert approx(values) == np.array([0.1])
+    assert approx(high_uncertainty) == np.array([9.48683493e-05])
+    assert approx(low_uncertainty) == np.array([9.48683493e-05])
+
+    assert approx(high_uncertainty[0]) == simple_efficiency_uncertainty(n1, n2) # 1e-6 relative error by default
+    assert approx(low_uncertainty[0]) == simple_efficiency_uncertainty(n1, n2) # 1e-6 relative error by default
+
+
+
+test_efficiency_simple_values()
```

Comment:
Do we need to call the function for the test to be tried by pytest? I do not think so.

Reply:
oops